internal/compiler-interface/src/main/java/xsbti/compile/IncrementalCompiler.java

```diff
@@ -67,5 +57,5 @@
 	* @param targetJar Where to create the output jar file containing the compiled classes.
 	* @param instance The ScalaInstance to compile the compiler interface for.
 	* @param log The logger to use during compilation. */
-	void compileInterfaceJar(String label, File sourceJar, File targetJar, File interfaceJar, ScalaInstance instance, Logger log);
+	// void compileInterfaceJar(String label, File sourceJar, File targetJar, File interfaceJar, ScalaInstance instance, Logger log);
```

Comment:
We need some mechanism to provide `AnalyzingCompiler`, otherwise this API is not useful.